src/password.js

```diff
@@ -15,6 +14,21 @@ function hash(password) {
   });
 }
 
+try {
+  bcrypt = require('bcrypt');
```

Comment:
here we override the `bcrypt` module altogether, so if it loads, the next `bcrypt.compare` will correctly call.

I had to do it that way for `hash` as both modules don't share the same signature.